ansible/roles/ml-data-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
+# tasks file for data-pipeline
+- name: Git checkout
+  git:
+    repo: 'https://github.com/Sunbird-Ed/ml-analytics-service.git'
+    dest: "/opt/sparkjobs/ml-analytics-service"
+    version: "{{ ml_analytics_version|default('release-4.0.0') }}"
```

Comment:
@sivakarthikeyanoptit remove the default value. We will always specify which version to deploy. else there are chances for erroneous deployments.

Reply:
done